monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/AvailablePlugins.js

```diff
@@ -79,8 +88,9 @@ const AvailablePlugins = () => {
   return (
     <Box>
       <SearchBar />
-      <BasePlugins plugins={availablePlugins}
-                   setPlugins={setAvailablePlugins}
+      <TypeFilter allPlugins={availablePlugins} setPlugins={setDisplayedPlugins}
```

Comment:
I've thought a bit about how to make it scalable, but since this PR doesn't need it, it's better to leave it the one who needs it (or I have to make another component for tests and then omit the changes)